src/components/Cards/Power/PowerCard.tsx

```diff
@@ -146,6 +146,24 @@ export default ({
                         }
                         disabled={disabled}
                     />
+                    <Toggle
+                        label={
+                            <DocumentationTooltip
+                                card={card}
+                                item="EnableVBatLow"
+                            >
+                                <>
+                                    <span>Enable Charging Below V</span>
```

Comment:
The only suggestion I might have is that this line might be a bit long for UI. So I might suggest something like:
```suggestion
                                    <span>Charging Below V</span>
```
Assuming that there is a toggle below this, we don't need Enable in the option name.